README.md

```diff
@@ -108,6 +110,28 @@ You can restrict what fields to send by adding `api: false` to a specific schema
 }
 ```
 
+Or you can require only specific fields in the GET request by adding the query filters `includeFields` and `excludeFields`.
+
+Examples:
+`/api/v1/products?includeFields=type,slug,name`
+`/api/v1/products?excludeFields=type,slug,name`
+
+It is useless to use both `includeFields` and `excludeFields` in the same query, as `includeFields` has the priority over `excludeFields`.
+
+Excluded fields with `api: false` or `api: 'editPermissionRequired'` if appropriate will not be displayed of course, even if added to `includeFields`.
+Example of a response to `/api/v1/products?includeFields=type,slug,excludedFieldInSchema`
```

Comment:
Something seems not right here. `excludedFieldInSchema` seems incorrect and the rest of the example is concrete with real fields you might use, which is good...

Reply:
I am not sure I understand your comment. This example shows that an excluded field (through `api: false` for instance) will not be sent though it was required in the query.
Do you want me to replace the example with a schema and then an example like the following?
```
     {
        name: 'price',
        type: 'float'
      },
      {
        name: 'tax',
        type: 'float',
      },
      {
        name: 'commission',
        type: 'float',
        api: false
      }
```

and then `/api/v1/products?includeFields=type,slug,price,commission` will output

```
[
  {
    _id: 'whatever_id',
    type: 'product',
    slug: 'product-key-product',
    price: 205.5
  }
]
```

Reply:
Or maybe, the example is not necessary because the feature is obvious..

Reply:
What do you think? @boutell